examples/js/loaders/GLTFLoader.js

```diff
@@ -1381,7 +1264,7 @@ THREE.GLTFLoader = ( function () {
 			return parser.getDependency( 'bufferView', accessor.bufferView ).then( function ( bufferView ) {
 
 				var itemSize = WEBGL_TYPE_SIZES[ accessor.type ];
-				var TypedArray = WEBGL_COMPONENT_TYPES[ accessor.componentType ];
+				var TypedArray = THREE.WEBGL_TO_THREE[ accessor.componentType ];
```

Comment:
Wouldn't it be better to use the same approach everywhere? `THREE.WebGLUtils.fromGL()`...

Reply:
fixed